components/src/core/components/TableSidebar/TableSidebar.vue

```diff
@@ -182,6 +182,9 @@ export default defineComponent({
     const toggleLeftPanel = () => {
       isLeftPanelOpen.value = !isLeftPanelOpen.value;
       emit('side-panel:onToggle', isLeftPanelOpen.value);
```

Comment:
Don't we have a way to listen for this event rather than having a browser event? @Super-Chama any idea?

Reply:
This vue custom event is only captured by immediate parent. vuex is another solution. we do have [mitt](https://github.com/developit/mitt) event library but since oxd and angular is separately packaged it won't work together.
since we are dealing with 2 libs browser event seems to be the common ground.

Reply:
I will keep the browser events. (Y)